tests/stack/test_api.py

```diff
@@ -158,7 +158,7 @@ def test_deploy_relative_paths(self, stack_api, tmpdir):
         with open(config_path, 'w+') as f:
             json.dump(TEST_STACK, f)
 
-        def _deploy_resource(resource, stack_status):
+        def _deploy_resource(resource, stack_status, **kwargs):
```

Comment:
Will you be adding tests with flag variations, e.g. overwrite-notebooks?

Reply:
Yes, that would be good to add. I'll get that going.

Reply:
Added tests